### PR TITLE
Add logging page

### PR DESCRIPTION
**Pull request by lloydmeta**

### CODE REVIEW

**Comment by mauhiz** [comment]
:+1:

File: source/logging-visualization.md.erb [hunk 1]
```diff
@@ -3,3 +3,115 @@ title: Logging & Visualization
 nav_order: 70
 ---
 
+## Pretty graphs
+
+Asides from performance and resilience benefits, routing backend services through Octoparts gives you the benefit of effortlessly* having the metrics for all of them in one place. Visualisations are avaiable in [Kibana](http://www.elasticsearch.org/overview/kibana/) and/or [Hystrix dashboard](https://github.com/Netflix/Hystrix/tree/master/hystrix-dashboard).
```
Comment (cb372) [inline source/logging-visualization.md.erb]: typo: available

Comment (lloydmeta) [inline source/logging-visualization.md.erb]: Thanks. Fixed.

File: source/logging-visualization.md.erb [hunk 2]
```diff
@@ -3,3 +3,115 @@ title: Logging & Visualization
...
+If you already know how to set up Fluentd, feel free to do it Your Way.
+
+1. Install the `td-agent` package
+2. Install the `libcurl-devel` package
+3. Install the `fluent-plugin-elasticsearch` gem: `$ fluent-gem install fluent-plugin-elasticsearch` **Note:** this is `fluent-gem install` _not_ `gem install`
+4. Create a temp log directory at `/etc/td-agent/tmp`
+5. Add a config Fluentd config file at `/etc/td-agent/td-agent.conf` with the following contents
+
+  ```
+  ## You will want to substitute your actual Octoparts log directory, ElasticSearch host, and
```
Comment (cb372) [inline source/logging-visualization.md.erb]: s/ElasticSearch/Elasticsearch/ everywhere please

Comment (lloydmeta) [inline source/logging-visualization.md.erb]: Done.

**Pull request merged**